src/app/_variables.scss

```diff
@@ -120,6 +120,68 @@ $search-bar-height: 28px;
   }
 }
 
+// Masterwork pattern a la .src/images/exotic-masterwork.png
+// Border with gradient running into top-right / bottom-left corners
+// Dark gradient from top, masterwork border gradient from bottom, striped diamond overlay for texture
+// TODO check whether rendering a bunch of gradients is less performant than slapping in a PNG
+@mixin masterwork($top-gradient: 0.05, $light-stripe: 0.05, $dark-stripe: 0.02) {
+  // Border
+  border-width: 1px;
+  border-style: solid;
+  border-image-slice: 1;
+  border-image-source: conic-gradient(
+    from 45deg at 50% 50%,
+    var(--theme-item-polaroid-masterwork-border) 0%,
+    transparent 24%,
+    transparent 25%,
+    var(--theme-item-polaroid-masterwork-border) 50%,
+    var(--theme-item-polaroid-masterwork-border) 50%,
+    transparent 74%,
+    transparent 75%,
+    var(--theme-item-polaroid-masterwork-border) 100%
+  );
+
+  // Overlay
+  $light-stripe: rgba(255, 255, 255, $light-stripe);
+  $dark-stripe: rgba(0, 0, 0, $dark-stripe);
+  background: linear-gradient(rgba(0, 0, 0, $top-gradient) 0%, transparent 50%),
+    linear-gradient(transparent 50%, var(--theme-item-polaroid-masterwork-border) 250%),
```

Comment:
```suggestion
    linear-gradient(transparent 50%, #eade8b 200%),
```
It might be worth experimenting with this gradient a bit more. In the default theme, `--theme-item-polaroid-masterwork-border` is white, which results in masterworked items looking a little washed out. Using a light yellow (e.g. `#eade8b` = `var(--theme-item-polaroid-masterwork)` in default theme) results in a warmer glow.

![dim-item-icon-masterwork-overlay-glow](https://github.com/DestinyItemManager/DIM/assets/17512262/f61da3f3-bc89-4433-99b0-95bccfe61af8)

Not sure what the expected behaviour is for non-default themes, but my initial thought is the gold glow would be theme-agnostic because player understand that gold = masterwork.

Reply:
Makes sense, but rather than hardcode the colour in the mixin, i think we can just modify the default theme to use the same golden masterwork border shade that's used in the dark theme `#ae791e` — this is the graded border that runs into the bottom left/top right corners of the image (as opposed to the the polaroid frame). 

![image](https://github.com/DestinyItemManager/DIM/assets/1396158/6e53bf18-3874-48fb-b230-41ce8b0e2289)

Reply:
I think the white graded border really sells the Masterwork "shininess", so would like to keep that if possible.

I pushed an experiment into the `item-tile-polaroid-theming` branch [here](https://github.com/DestinyItemManager/DIM/commit/51aeb67f69b27e7539dad78324ff2ab89f6a31f9) which might be of some use.

![dim-item-icon-masterwork-styles](https://github.com/DestinyItemManager/DIM/assets/17512262/482bc481-933f-4abd-af62-8ddd18d138ec)


There's less contrast between masterwork & non-masterwork polaroids, but I think the overlay meshes with the polaroid border a little better.